script.js

```diff
@@ -0,0 +1,178 @@
+    options: [
+      { text: "Musa", isCorrect: false },
+      { text: "Issa", isCorrect: false },
+      { text: "Muhammad", isCorrect: true },
+      { text: "Ibrahim", isCorrect: false },
+    ],
+    explanation: "The last prophet in Islam is Prophet Muhammad (peace be upon him).",
+  },
+  {
+    id: 3,
+    question: "What is the primary place of worship for Muslims?",
+    options: [
+      { text: "Church", isCorrect: false },
+      { text: "Synagog", isCorrect: false },
+      { text: "Mosque", isCorrect: true },
+      { text: "Temple", isCorrect: false },
+    ],
+    explanation: "The primary place of worship for Muslims is a mosque.",
+  },
+];
+
+let player1Score = 0;
+let player2Score = 0;
+let currentPlayer = 1;
+let currentQuestionIndex = 0;
+
+const questionTitle = document.querySelector('.questionTitle');
+const optionsContainer = document.querySelector('.options');
+const form = document.querySelector('#quizForm');
+const questionList = document.querySelector('.questionList');
+const player1ScoreHtml = document.getElementById('player1-score');
+const player2ScoreHtml = document.getElementById('player2-score');
+const currentTurnHtml = document.getElementById('current-turn');
+const playerForm = document.getElementById('playerForm');
+const player1UI = document.getElementById('player1UI');
+const player2UI = document.getElementById('player2UI');
+
+
+playerForm.addEventListener("submit", function(event){
+  event.preventDefault();
+  const player1ValueName = document.getElementById('player1ValueName').value;
+  const player2ValueName = document.getElementById('player2ValueName').value;
+
+  player1UI.textContent = player1ValueName
+  player2UI.textContent = player2ValueName
+
+  playerForm.classList.add('hidden');
+});
+
+
+function displayQuestion() {
+  const currentQuestion = quizQuestions[currentQuestionIndex];
+  questionTitle.textContent = currentQuestion.question;
+
+  const optionElements = currentQuestion.options.map(
+    option => `
+    <label>
+      <input type="radio" name="answer" value="${option.text}">
+      ${option.text}
+    </label>
+  `
+  );
+  optionsContainer.innerHTML = optionElements.join('');
+}
+
+function submitForm(event) {
+  event.preventDefault();
+  const selectedOption = document.querySelector('input[name="answer"]:checked');
+  if (!selectedOption) {
+    alert('Please select an answer.');
+    return;
+  }
+
+  const currentQuestion = quizQuestions[currentQuestionIndex];
+  const isCorrect = currentQuestion.options.find(
+    option => option.text === selectedOption.value
+  ).isCorrect;
+
+  if (isCorrect) {
+    alert(`Correct answer. Explanation: ${currentQuestion.explanation}`);
+    if (currentPlayer === 1) {
+      player1Score++;
+      player1ScoreHtml.textContent = player1Score;
+    } else {
+      player2Score++;
+      player2ScoreHtml.textContent = player2Score;
+    }
+  } else {
+    alert(`Wrong answer. Explanation: ${currentQuestion.explanation}`);
+  }
```

Comment:
The assignment asks that when a player gets a question wrong, the other player gets a point.